public/js/monitor.js

```diff
@@ -36,7 +36,7 @@ function isValidEmail(val) {
 }
 
 
-function doOauth(el) {
+function doOauth(el, {emailWatch = false} = {}) {
```

Comment:
whoa.

Reply:
Would it be easier to just make it the following? 
```
doOauth(el, emailWatch = fase) {…
```